### PR TITLE
Leeben13

### PR DESCRIPTION
**Pull request by 20leebe**
I changed the timeStamp from Event from a map to LocalDateTime and changed addMedicineWindow to reflect these changes.

### CODE REVIEW

File: src/main/java/interface_adapters/AppManager.java [hunk 1]
```diff
@@ -52,7 +58,7 @@ public void run(Map<String, Window> windows){
     /**
      * Shows the start screen window.
      */
-    public void showStartScreenWindow(){
+    public void showStartScreenWindow() throws ParseException {
```
Comment (SujoyNath1511) [inline src/main/java/interface_adapters/AppManager.java:61]: Not completely sure, but why do all of these methods throw ParseException?

File: src/main/java/frameworks_and_drivers/Main.java [hunk 2]
```diff
@@ -11,7 +13,7 @@
     /**
      * The main method.
      */
-    public static void main(String[] args) {
+    public static void main(String[] args) throws ParseException {
```
Comment (SujoyNath1511) [inline src/main/java/frameworks_and_drivers/Main.java:16]: Where do you handle the ParseException?

File: src/main/java/frameworks_and_drivers/AddMedicineWindow.java [hunk 3]
```diff
@@ -61,10 +61,9 @@ public AddMedicineWindow(Scanner scanner) {
         String[] times = new String[howManyTimesInt];
         String time = " ";
         for (int i = 0; i < howManyTimesInt; i++){
-            while(!isNumeric(time) || 0 > Double.parseDouble(time) || 24 <= Double.parseDouble(time)){
-                System.out.println("For the "+ (i + 1) + "'st time, pick a time (In the form of XX.XX." +
-                        "Note that  such as 10:30 becomes " +
-                        "10.5, or 13:45 as 13.75");
+            while(!isValidTime(time)){
+                System.out.println("For the "+ (i + 1) + "'st time, pick a time (In the form of XX:XX. Use the 24 hour " +
```
Comment (SujoyNath1511) [inline src/main/java/frameworks_and_drivers/AddMedicineWindow.java:65]: This is good!

File: src/main/java/entities/Schedule.java [hunk 4]
```diff
@@ -58,18 +58,18 @@ public boolean removeEvent(Event event){
      * @return  A string representation of the Schedule.
      */
     @Override
+    /* TODO Make it so that it sorts the schedule*/
```
Comment (SujoyNath1511) [inline src/main/java/entities/Schedule.java:61]: Probably, need to change the sortEvents method.

File: src/main/java/interface_adapters/AppManager.java (hunk 1 above)
Comment (20leebe) [reply src/main/java/interface_adapters/AppManager.java:57]: The code was giving me an error and said I needed to add it.

File: src/main/java/frameworks_and_drivers/Main.java (hunk 2 above)
Comment (20leebe) [reply src/main/java/frameworks_and_drivers/Main.java:16]: I still need to look into this. I'm not sure why it's telling me to add this.

**Pull request closed without merging**